src/App.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Routes, Route } from 'react-router-dom';
+
+import './App.css';
+import CountriesList from './components/CountriesList';
+import CountryDetails from './components/CountryDetails';
+import RegionsList from './components/RegionsList';
+import RegionDetails from './components/RegionDetails';
+
+function App() {
+  const { casesByCountry } = useSelector((state) => state);
+  return (
+    <Routes>
+      <Route path="/" element={<CountriesList />} />
+      {
+        Object.keys(casesByCountry).map(
+          (country) => (
+            <Route
+              key={country}
+              path={`/${country}`}
+              element={
+                casesByCountry[country].regions.length === 0
+                  ? <CountryDetails country={casesByCountry[country]} />
+                  : <RegionsList country={casesByCountry[country]} />
+              }
+            />
+          ),
+        )
+      }
+      {
+        Object.keys(casesByCountry).map(
+          (country) => (
+            casesByCountry[country].regions.map(
+              (region) => (
+                <Route
+                  key={region.id}
+                  path={`/${country}/${region.id}`}
+                  element={<RegionDetails region={region} />}
+                />
+              ),
+            )
+          ),
+        )
+      }
+    </Routes>
+  );
+}
```

Comment:
- [ ] your project is looking really good. please adjust the design so that it fits on any mobile size currently your design has a fixed width and is not flexible. the project only requires a mobile design although it should still be flexible and fit in different mobile sizes and not have a fixed width. please address this issue